modules/hcp-proxy-cp-workloads.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-networking.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-proxy-cp-workloads_{context}"]
+= Control plane workloads that need to access external services
+
+Operators that run in the control plane need to access external services through the proxy that is configured for the hosted cluster. The proxy is usually accessible only through the data plane. The control plane workloads are as follows:
+
+* The Control Plane Operator needs to validate and obtain endpoints from certain identity providers when it creates the OAuth server configuration.
+
+* The OAuth server needs non-LDAP identity provider access.
+
+* The OpenShift API server handles image registry metadata import.
+
+* The Ingress Operator needs access to validate external canary routes.
+
+In a hosted cluster, traffic that originates from the Control Plane Operator, Ingress Operator, OAuth server, and OpenShift API server pods should be sent through the data plane to the configured proxy and then to its final destination. 
```

Comment:
```suggestion
In a hosted cluster, traffic that originates from the Control Plane Operator, Ingress Operator, OAuth server, and OpenShift API server pods must be sent through the data plane to the configured proxy and then to its final destination. 
```

- If this is something done by users, then can be converted into an active voice.  --   "...., you must send traffic that....through the data..."
- Avoid using should
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed.

Reply:
Both good points. Will fix.